src/main/java/se/sundsvall/casestatus/integration/opene/OpenEClient.java

```diff
@@ -1,87 +1,30 @@
 package se.sundsvall.casestatus.integration.opene;
 
-import java.io.IOException;
-import java.net.URI;
-import java.time.LocalDate;
-import java.time.ZoneId;
-import java.util.Map;
+import static se.sundsvall.casestatus.integration.opene.configuration.OpenEIntegrationConfiguration.CLIENT_ID;
 
-import org.apache.hc.client5.http.auth.AuthScope;
-import org.apache.hc.client5.http.auth.UsernamePasswordCredentials;
-import org.apache.hc.client5.http.classic.methods.HttpGet;
-import org.apache.hc.client5.http.impl.auth.BasicCredentialsProvider;
-import org.apache.hc.client5.http.impl.classic.HttpClients;
-import org.apache.hc.core5.http.io.entity.EntityUtils;
-import org.springframework.stereotype.Component;
-import org.springframework.web.util.UriComponentsBuilder;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 
-import se.sundsvall.casestatus.integration.opene.exception.OpenEException;
+import se.sundsvall.casestatus.integration.opene.configuration.OpenEIntegrationConfiguration;
 
-@Component
-public class OpenEClient {
+import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
 
-	private static final String ERRANDS_PATH = "/api/instanceapi/getinstances/family/{familyid}";
-	private static final String PDF_PATH = "/api/instanceapi/getinstance/{flowinstanceid}/pdf";
-	private static final String ERRAND_PATH = "/api/instanceapi/getinstance/{flowinstanceid}/xml";
-	private static final String STATUS_PATH = "/api/instanceapi/getstatus/{flowinstanceid}";
+@FeignClient(name = CLIENT_ID, url = "${integration.open-e.base-url}", configuration = OpenEIntegrationConfiguration.class)
+@CircuitBreaker(name = CLIENT_ID)
+public interface OpenEClient {
+	String TEXT_XML_CHARSET_ISO_8859_1 = "text/xml; charset=ISO-8859-1";
 
-	private static final String FLOW_INSTANCE_ID_ATTRIBUTE_NAME = "flowinstanceid";
-	private static final String FAMILY_ID_ATTRIBUTE_NAME = "familyid";
+	@GetMapping(path = "/api/instanceapi/getinstances/family/{familyid}", consumes = TEXT_XML_CHARSET_ISO_8859_1, produces = TEXT_XML_CHARSET_ISO_8859_1)
```

Comment:
Does this resource really consumes XML (i.e. does it have a body)?

(comment applicable to all GetMappings in this interface)